clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2659,8 +2659,9 @@ An error will be given if:
   - Specified values violate subtarget specifications;
   - Specified values are not compatible with values provided through other
     attributes;
-  - The AMDGPU target backend is unable to create machine code that can meet the
-    request.
+
+The AMDGPU target backend will emit a warning whenever it is unable to
```

Comment:
This looks indented differently, without the list - as the previous comment did

Reply:
Do you want me to format it like this instead to mimic the previous formatting?
```
A warning will be given if:
  - ...
```

Reply:
Oh, I wasn't looking at the header bit here. I was also assuming this is using the sphinx formatting rules? I suppose it doesn't really matter